### ISSUE TITLE
[Bug] Fenix not working on HTC Nexus 9 Tablet

### ISSUE THREAD

**Issue by AndiAJ** [mozilla-mobile/fenix#3421]
## Steps to reproduce
Install the latest Fenix version (2019-06-13) from Trackcluster.
Launch the app

### Expected behavior
The app is launched properly and can be used

### Actual behavior
The app isn't launched , white screen

### Device information
* Android device: HTC Nexus 9 Android 7.1.1
* Fenix version: 1.0.194(Build #11640609GV)

### Notes
Logcat: [Nexus9.txt](https://github.com/mozilla-mobile/fenix/files/3286067/Nexus9.txt)
Video: https://drive.google.com/file/d/1by-TffSWDaka7naXLL7Jyc_3VKXL9MUi/view?usp=sharing

**Comment by sv-ohorvath** [comment]
Not working since: 5/24. See also: #2999.

**Comment by jonalmeida** [comment]
Relevant part of the logcat:
```
2019-06-13 15:32:15.843 4202-4257/? W/GeckoLinker: /data/app/org.mozilla.fenix-1/lib/arm/libxul.so: unhandled flags #8 not handled
2019-06-13 15:32:15.844 4202-4257/? W/GeckoLinker: /data/app/org.mozilla.fenix-1/lib/arm/liblgpllibs.so: unhandled flags #8 not handled
2019-06-13 15:32:15.848 4202-4202/? E/KeyFrames: unable to load
    java.lang.NoSuchMethodException: <init> []
        at java.lang.Class.getConstructor0(Class.java:2204)
        at java.lang.Class.getConstructor(Class.java:1683)
        at androidx.constraintlayout.motion.widget.KeyFrames.<clinit>(KeyFrames.java:2)
        at androidx.constraintlayout.motion.widget.MotionScene.<init>(MotionScene.java:18)
        at androidx.constraintlayout.motion.widget.MotionLayout.init(MotionLayout.java:6)
        at androidx.constraintlayout.motion.widget.MotionLayout.<init>(MotionLayout.java:60)
        at java.lang.reflect.Constructor.newInstance0(Native Method)
        at java.lang.reflect.Constructor.newInstance(Constructor.java:430)
        at android.view.LayoutInflater.createView(LayoutInflater.java:645)
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:787)
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:727)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:858)
        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:821)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:518)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:426)
        at org.mozilla.fenix.home.HomeFragment.onCreateView(HomeFragment.kt:2)
        at androidx.fragment.app.Fragment.performCreateView(Fragment.java:4)
        at androidx.fragment.app.FragmentManagerImpl.moveToState(FragmentManagerImpl.java:88)
        at androidx.fragment.app.FragmentManagerImpl.addAddedFragments(FragmentManagerImpl.java:6)
        at androidx.fragment.app.FragmentManagerImpl.executeOpsTogether(FragmentManagerImpl.java:54)
        at androidx.fragment.app.FragmentManagerImpl.removeRedundantOperationsAndExecute(FragmentManagerImpl.java:11)
        at androidx.fragment.app.FragmentManagerImpl.execPendingActions(FragmentManagerImpl.java:4)
        at androidx.fragment.app.FragmentManagerImpl.dispatchStateChange(FragmentManagerImpl.java:4)
        at androidx.fragment.app.Fragment.performActivityCreated(Fragment.java:9)
        at androidx.fragment.app.FragmentManagerImpl.moveToState(FragmentManagerImpl.java:99)
        at androidx.fragment.app.FragmentManagerImpl.moveFragmentToExpectedState(FragmentManagerImpl.java:9)
        at androidx.fragment.app.FragmentManagerImpl.moveToState(FragmentManagerImpl.java:210)
        at androidx.fragment.app.FragmentManagerImpl.dispatchStateChange(FragmentManagerImpl.java:2)
        at androidx.fragment.app.FragmentActivity.onStart(FragmentActivity.java:9)
        at androidx.appcompat.app.AppCompatActivity.onStart(AppCompatActivity.java:1)
        at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1248)
        at android.app.Activity.performStart(Activity.java:6696)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2628)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2726)
        at android.app.ActivityThread.-wrap12(ActivityThread.java)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1477)
        at android.os.Handler.dispatchMessage(Handler.java:102)
        at android.os.Looper.loop(Looper.java:154)
        at android.app.ActivityThread.main(ActivityThread.java:6119)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.
```

**Comment by AndiJoltan** [comment]
Hi, still happening on Firefox Preview v1.0.0-rc.1

Thanks,
Andi

**Comment by sv-ohorvath** [comment]
dupe and fixed on #2945.
